documentation/docs/30-advanced/15-websockets.md

```diff
@@ -0,0 +1,126 @@
+---
+title: WebSockets
+---
+
+## The `socket` object
+
+[WebSockets](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) provide a way to open a bidirectional communication channel between the client and server.
+
+SvelteKit accepts a `socket` object in `+server.js` files that you can use to handle WebSocket connections to different routes in your app.
```

Comment:
Following the wording from the `actions` docs https://svelte.dev/docs/kit/form-actions
```suggestion
A `+server.js` file can export a `socket` object to handle WebSocket connections.
```